test/clitest/common_windows.go

```diff
@@ -0,0 +1,15 @@
+//go:build windows
+
+package clitest
+
+var (
+	AddrArg         = "-namedPipeName"
+	AddrError       = "rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: open \\\\\\\\.\\\\pipe\\\\does-not-exist: The system cannot find the file specified.\"\n"
```

Comment:
nit: for simplification can you update this string to use "`"?

Reply:
Sure, it was copy-pasted from another file that I moved here, but this is a good improvement for readability.

Reply:
Actually we cannot make this a raw string literal because it has an escape sequence at the end, which will not be interpreted correctly if we use a raw string literal.